src/views/CardStack.js

```diff
@@ -36,6 +38,8 @@ import TransitionConfigs from './TransitionConfigs';
 
 const emptyFunction = () => {};
 
+const windowSize = Dimensions.get('window');
```

Comment:
I think we may be able to use `props.layout` instead of `Dimensions` for this. Generally Dimensions is a hack because the screen size may theoretically change over time (and windowSize will get stale). Or theoretically the card stack may not take up the full screen and this would be wrong